tracker/src/main/java/org/matomo/sdk/dispatcher/PacketFactory.java

```diff
@@ -37,19 +37,11 @@ public PacketFactory(final String apiUrl) {
     public List<Packet> buildPackets(final List<Event> events) {
         if (events.isEmpty()) return Collections.emptyList();
 
-        if (events.size() == 1) {
-            Packet p = buildPacketForGet(events.get(0));
-            if (p == null) return Collections.emptyList();
-            else return Collections.singletonList(p);
-        }
-
         int packets = (int) Math.ceil(events.size() * 1.0 / PAGE_SIZE);
         List<Packet> freshPackets = new ArrayList<>(packets);
         for (int i = 0; i < events.size(); i += PAGE_SIZE) {
             List<Event> batch = events.subList(i, Math.min(i + PAGE_SIZE, events.size()));
-            final Packet packet;
-            if (batch.size() == 1) packet = buildPacketForGet(batch.get(0));
```

Comment:
@d4rken 
When I understand right, you did in 1f8ccd6253b3e9f29cc84d4c45587931a62a10c8 this decision if it's a `get` or a `post` by asking `batch.size()`, am I right ?

Do you agree with this changes ?

Reply:
I don't remember why I did it this way :thinking:.

Might have been an optimization, i.e. `GET` having less overhead than `POST` for single events. In hindsight I don't think the overhead would be huge enough to warrant both GET and POST.

I'm neutral on this change.

@nostromoo Please consider adding some descriptions to your PRs. What made you change this?